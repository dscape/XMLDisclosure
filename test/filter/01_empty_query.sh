#!/bin/sh
################################################################################
##                                                                            ##
##  Unit Test                                                                 ##
##    Author: Nuno Job  ( nunojobpinto at gmail )                             ##
##    Date: August 2009                                                       ##
##    License: creativecommons.org/licenses/by/3.0                            ##
##                                                                            ##
################################################################################
db2 connect to $1 > /dev/null
db2 "call xmldisclosure.filter('', 'contact')"
