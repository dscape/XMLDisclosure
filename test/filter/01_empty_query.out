SQL4302N  Procedure or user-defined function "XMLDISCLOSURE.FILTER", specific 
name "FILTER" aborted with an exception "Invalid XQuery statement: null, 
doesn't st".  SQLSTATE=38501
